dev/fattest.simplicity/src/com/ibm/websphere/simplicity/CDIArchiveHelper.java

```diff
@@ -69,6 +70,16 @@ public static WebArchive addBeansXML(WebArchive webArchive, Asset beans) {
         return webArchive.addAsWebInfResource(beans, BEANS_XML);
     }
 
+    /**
+     * Create a WEB-INF/beans.xml file in a war
+     *
+     * @param webArchive The WAR to create the beans.xml in
+     * @param srcPackage The package where the source beans.xml file can be found
+     */
+    public static WebArchive addBeansXML(WebArchive webArchive, Package srcPackage) {
+        return webArchive.addAsWebInfResource(srcPackage, BEANS_XML, BEANS_XML);
+    }
```

Comment:
It's possibly nicer to pass the class in here rather than the package.

I'm actually tempted to suggest getting rid of this method and recommend that tests do
```
addBeansXml(war, MyTestServlet.class.getResource("beans.xml"));
```

This would make the method slightly less magic.

Reply:
These are convienence utility methods designed to reduce repetitious typing. I have added a couple of extra versions of the method; One which takes the "owning" class and the other which takes a resource URL ... so you are free do it in whichever way suits.